x/examples/outline-cli/routing_linux.go

```diff
@@ -113,6 +113,8 @@ func setupIpRule(svrIp string, routingTable, routingPriority int) error {
 	ipRule.Invert = true
 
 	if err := netlink.RuleAdd(ipRule); err != nil {
+                // assuming duplicate from previous run, just to make sure it does not stays stale forever in the routing table
+		defer cleanUpRule()
```

Comment:
Do we actually need the `defer` call? Why not just calling `cleanUpRule()` directly?